src/main/kotlin/uk/gov/justice/digital/hmpps/prisonerfromnomismigration/alerts/AlertsSynchronisationService.kt

```diff
@@ -191,36 +192,40 @@ class AlertsSynchronisationService(
     val bookingId = prisonerMergeEvent.additionalInformation.bookingId
     val offenderNo = prisonerMergeEvent.additionalInformation.nomsNumber
     val removedOffenderNo = prisonerMergeEvent.additionalInformation.removedNomsNumber
-    val alerts = nomisApiService.getAlertsByBookingId(bookingId).alerts
-    val alertsMapping = alerts.groupBy { mappingApiService.getOrNullByNomisId(bookingId = it.bookingId, alertSequence = it.alertSequence) }
-    val newAlerts = alertsMapping[null] ?: emptyList()
-    val retainedDpsAlertIds = alertsMapping.keys.filterNotNull().map { it.dpsAlertId }
```

Comment:
meat of change, switch to different DPS endpoint and Mapping endpoint similar to how we handle new bookings etc